### ISSUE TITLE
GeoIP for NPM

### ISSUE THREAD

**Issue by mxm199** [NginxProxyManager/nginx-proxy-manager#3334]
Hello!
This topic has already been raised in discussions, but a lot of water has passed under the bridge since 2019. I would like to know if it is still possible to connect the geoip module to the npm container? Even if using databases in the old format, enthusiasts [post them](https://mailfud.org/geoip-legacy).
Or perhaps there are other options for using IP access restrictions in the use case of Ubuntu Server + docker + npm container?
Thank you in advance for your cooperation!

**Comment by ale82x** [comment]
yes , i also intersted on geoip to block some country or open some hosts only from my country.

thank you

**Comment by maxdd** [comment]
although maybe secondary to this, have a look at 

https://github.com/smilebasti/npmGrafStats

and 

https://github.com/mitchellkrogza/nginx-ultimate-bad-bot-blocker

i've integrated both and im quite satisfied
